forti_token_client.go

```diff
@@ -38,6 +40,8 @@ type fortiTokenClient struct {
 }
 
 func (c *fortiTokenClient) newGetRequest(url string) (*http.Request, error) {
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
Can you make this a flag setting?
In my environment all firewalls have correct certificates, so running this is a security issue. However, I can certainly see what some folks would like to be able to skip this check.